business/iter8.go

```diff
@@ -170,6 +171,35 @@ func (in *Iter8Service) ParseJsonForCreate(body []byte) (string, error) {
 	object.Spec.TrafficControl.MaxTrafficPercentage = newExperimentSpec.TrafficControl.MaxTrafficPercentage
 	object.Spec.TrafficControl.MaxIterations = newExperimentSpec.TrafficControl.MaxIterations
 	object.Spec.TrafficControl.TrafficStepSize = newExperimentSpec.TrafficControl.TrafficStepSize
+	object.Spec.Analysis.AnalyticsService = "http://iter8-analytics.iter8:" + strconv.Itoa(in.k8s.GetAnalyticPort())
```

Comment:
Is this `object.Spec.Analysis.AnalyticsService` used just for informational usecases with the user ?
Or is there any planned query trom Kiali to this service ?
Also this url seems perhaps it needs to be in the config.go (Iter8 section) and probably in the operator as defaults.
I can take care of the operator to simplify the work.

Reply:
Ok, I can respond to myself, so, it's just to show this info to the user as data but no logic involved.